src/types/index.ts

```diff
@@ -32,10 +32,20 @@ export type CollectionRecords = readonly CollectionRecord[]
  * ***********************
  * */
 
-/* Bookmark refers to a user-created object containing a url */
+export type WidgetType = 'Collection' | 'News'
+
+export type Widget = {
+  _id: string | ObjectId
+  title: string
+  type: WidgetType
+}
+
+export type MySpaceWidget = Widget | Collection
+export type MySpace = MySpaceWidget[]
 
+/* Bookmark refers to a user-created object containing a url */
 export type Bookmark = {
-  _id: string
+  _id: string | ObjectId
```

Comment:
@abbyoung so I started doing what we talked about and converting `_id` type to `ObjectId` everywhere, but it was becoming a lot of changes in an already big PR so I decided to stop and created a [separate story](https://app.shortcut.com/orbit-truss/story/303/use-objectid-consistently-instead-of-string-for-mongodb-object-ids) for it instead

Reply:
Makes sense! Will be easier to change and test all at once. Thanks!